sources/packages/backend/libs/test-utils/src/factories/student.ts

```diff
@@ -15,7 +15,7 @@ export function createFakeStudent(
   student.birthDate =
     options?.initialValue?.birthDate ??
     getISODateOnlyString(faker.date.past(18));
-  student.gender = "X";
+  student.gender = "nonBinary";
```

Comment:
As a suggestion, the equivalent would be "Prefer not to answer".

Reply:
Non binary still maps to 'X'. The equivalent for Prefer not to answer would be a blank.